app/services/notifications/new_mention/send.rb

```diff
@@ -0,0 +1,48 @@
+module Notifications
+  class Send
+
+    delegate :user_data, to: Notifications
+    delegate :comment_data, to: Notifications
+    attr_reader :mention
+
+    def initialize(mention)
+      @mention = mention
+    end
+
+    def self.call(*args)
+      new(*args).call
+    end
+
+    def call
+      Notification.create(
+        user_id: mention.user_id,
+        notifiable_id: mention.id,
+        notifiable_type: "Mention",
+        action: nil,
+        json_data: json_data
+      )
+    end
+
+    private 
+
+    def json_data
+      if mention.mentionable_type == "Comment"
+        return {
+          comment: comment,
```

Comment:
I suppose calling ` comment_data(mention.mentionable)` and `user_data(mention.user_id)` instead of having `comment` and `user` methods would make the code easier to understand.